src/ripple/peerfinder/impl/Logic.h

```diff
@@ -401,18 +406,15 @@ class Logic
             return Result::full;
         }
 
-        // Set key and cluster right before adding to the map
-        // otherwise we could assert later when erasing the key.
-        counts_.remove (*slot);
+        // Set the key right before adding to the map, otherwise we might
+        // assert later when erasing the key.
         slot->public_key (key);
-        slot->cluster (cluster);
-        counts_.add (*slot);
-
-        // Add the public key to the active set
-        auto const result = keys_.insert (key);
-        // Public key must not already exist
-        assert (result.second);
-        (void) result.second;
+        {
+            auto const result = keys_.insert(key);
```

Comment:
Do we enforce "one space between function/macro name and parenthesized argument list"?

Reply:
We aren’t super-strict about it. Our “space after” policy is a bit unclear and some files contain a mix of several iterations of the “preferred” style.